dataland-frontend/src/components/forms/parts/fields/NaceCodeFormField.vue

```diff
@@ -40,9 +40,27 @@ export default defineComponent({
   },
   data() {
     return {
-      selectedNaceCodes: [] as Array<string>,
+      innerSelectedNaceCodes: [] as Array<string>,
     };
   },
-  props: FormFieldProps,
+  computed: {
+    selectedNaceCodes: {
+      get(): [] {
+        return this.selectedNaceCodesBind ?? this.innerSelectedNaceCodes;
+      },
+      set(newValue: []) {
+        this.$emit("update:selectedNaceCodesBind", newValue);
+        this.innerSelectedNaceCodes = newValue;
+      },
+    },
+  },
+
+  props: {
+    ...BaseFormFieldProps,
+    selectedNaceCodesBind: {
+      type: Array,
+      required: false,
```

Comment:
I think it is "false" per default, so you could also remove it.

Same for `PercentageFormField.vue`

Reply:
👍